### DISCUSSION TITLE
How to Export data from cloud and Import to Localhost?

### DISCUSSION THREAD

**Discussion by mayurjobanputra** [danny-avila/LibreChat#4668] [Help Wanted]
I have been running Librechat on Railway for several months with a tonne of chats. How do I export my data and import it into my locally hosted version? 

Keep in mind: I'm technical, but not an expert Eng.

**Comment by alvieirajr** [comment]
I would start with backup the remote mongodb and restore to localhost instance of mongodb. Change the .env file to point to local instance of mongodb.
